src/test/java/io/openshift/booster/OpenshiftIT.java

```diff
@@ -43,36 +45,35 @@
     @RouteURL("${app.name}")
     private URL url;
 
+    private RequestSpecification requestSpecification;
+
     @Before
     public void setup() {
-        await().atMost(5, TimeUnit.MINUTES).until(() -> {
-            try {
-                return get(url).getStatusCode() == 200;
-            } catch (Exception e) {
-                return false;
-            }
-        });
```

Comment:
If this is removed, should add `@AwaitRoute`. (It's opt-in, not opt-out.)

Reply:
Updated. This makes me think that why `AwaitRoute` is not set by default with defaults and the user is the one who can decide to customize everything with `@AwaitRoute` or `@DisableAwaitRoute`?

Reply:
Because of https://github.com/arquillian/arquillian-cube/issues/837, essentially.

Reply:
Great, then maybe we can merge this PR and start thinking about making automatic. What I see is that probably in all cases you want to wait, and even the defaults might work in most of the cases.